backend/tests/test_api_errors.py

```
@@ -74,6 +74,93 @@ def raise_detail_dict_error() -> None:
         )
         self.assertNotIn(test_token, response.text)
 
+    def test_entitlement_denial_audit_events_are_allowlisted(self) -> None:
+        raw_token = "raw-token-should-not-render"
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to: "raw_token"

(S105)